tests/ui/conftest.py

```diff
@@ -1,8 +1,52 @@
+"""Configuration fixtures and functions for pytest."""
+
+import sys
+
 import pytest
 
+from pages.discopane import DiscoveryPane
+
 
 @pytest.fixture
 def capabilities(capabilities):
-    # In order to run these tests in Firefox 48, marionette is required
+    """Set Firefox capabilities."""
     capabilities['marionette'] = True
+    capabilities['acceptInsecureCerts'] = True
     return capabilities
+
+
+@pytest.fixture
+def my_base_url(base_url, firefox):
+    """Create custom base url depending on os and firefox version."""
+    ff_version = firefox.browser.firefox_version
+    os = sys.platform
+    return base_url + '/discovery/pane/{0}.0/{1}/normal'.format(ff_version, os)
+
+
+@pytest.fixture
+def open_discopane(my_base_url, selenium):
+    """Open discopane via the menu."""
+    with selenium.context(selenium.CONTEXT_CHROME):
+        toolbar = selenium.find_element_by_id('menu_ToolsPopup')
+        toolbar.click()
+        addons = selenium.find_element_by_id('menu_openAddons')
+        addons.click()
+    return DiscoveryPane(selenium, my_base_url).wait_for_page_to_load()
+
+
+@pytest.fixture
+def discovery_pane(selenium, my_base_url):
+    """Open the discovery pane via the URL."""
+    return DiscoveryPane(selenium, my_base_url).open()
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    """Configure Firefox preferences and additonal arguments."""
+    firefox_options.set_preference('extensions.webservice.discoverURL', 'https://discovery.addons.mozilla.org/%LOCALE%/firefox/discovery/pane/%VERSION%/%OS%/%COMPATIBILITY_MODE%')  # noqa
```

Comment:
Why isn't this URL set to `https://example.com:4000`? I'm confused

Reply:
This is to test that if the URL is wrong, discopane will not load. Kind of an edge case but I thought it was useful.

Reply:
Actually, my mistake. It appears that creating a custom profile with selenium does not populate this field. Maybe this isn't needed however.